### DISCUSSION TITLE
Does the offical support from github for education still available ?

### DISCUSSION THREAD

**Discussion by PingWang28** [community/community#123951] [GitHub Education]
### Hi everyone,

The deadline comes from Github for me to reverify my education information will be reached soon.  I try to public the issue I meet in this community also open a ticket as the guide from github. But almost half a month has passed , never get a valid offical support until now. I observer the status of many opened disscussion here is also like this.  
So does anyone know how the offcial support worked here  ?

**Comment by PingWang28** [comment]
pop , hope some can see this .

**Comment by namith1003** [comment]
if u can tell me the issue i could help you out, i also recently just revalidated my education package it actually was expired. u can still reupdate it after it expires

**Reply by PingWang28** [reply]
Thank you,  but I'm not expired yet . When I try to reverify my status, it always enter into a wrong link(https://github.com/pricing) which can't fill my information .

**Comment by johan-naizu** [comment]
you can reverify and keep your github student pack benefits as long as you are a student.

**Reply by PingWang28** [reply]
I'm a teacher , I just want to keep my benefits as a tearcher , there must be something wrong......

**Reply by johan-naizu** [reply]
https://education.github.com/discount_requests/application
you should be able to apply again from here.
Make sure you're logged in

**Reply by PingWang28** [reply]
no, if you are in the reverify stage, actually you can't enter into this page , it will be redirect to other page.

**Reply by johan-naizu** [reply]
has your education pack expired already?

**Reply by PingWang28** [reply]
No,  that's the reason I stucked . Finally, my ticket for github support get feedback ,  they let me education pack expired directly . I can do validation again as a fresh. That's some slowly , but work now . Thank you trace and help me
